mixer/pkg/config/mcp/backend.go

```diff
@@ -0,0 +1,284 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package mcp
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"sync"
+
+	"github.com/gogo/protobuf/proto"
+	"google.golang.org/grpc"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"istio.io/istio/galley/pkg/mcp/configz"
+	"istio.io/istio/galley/pkg/metadata/kube"
+
+	"istio.io/istio/galley/pkg/kube/converter/legacy"
+
+	mcp "istio.io/api/mcp/v1alpha1"
+	"istio.io/istio/galley/pkg/mcp/client"
+	"istio.io/istio/mixer/pkg/config/store"
+	"istio.io/istio/pkg/log"
+	"istio.io/istio/pkg/probe"
+)
+
+var scope = log.RegisterScope("mcp", "Mixer MCP client stack", 0)
```

Comment:
The MCP library itself doesn't add a scope, does it?

Reply:
No, the client library doesn't. (There is one in server though, but it doesn't affect Mixer).